app/models/event_dispatcher.rb

```diff
@@ -14,9 +14,13 @@ def dispatch!(user)
     return if triggered || !event_receiver.trigger_condition_met?
 
     # Run each of the event actions corresponding to the job
-    job.event_actions.each { |action| action.run!(user) }
+    event_receiver.event_actions.each { |action| action.run!(user) }
 
     # Set the record to be triggered
     self.triggered = true
   end
+
+  def reset_trigger!
```

Comment:
@davepagurek do you want me to keep this method, or remove it - since we're no longer calling it through the `IntervalWorker` job.

Reply:
Think we can take it out now